docs/architecture/oracles/oracles.mdx

```diff
@@ -21,7 +21,7 @@ and the blockchain. These infrastructure changes allow the Helium Network to eff
 new Networks as Decentralized Network Protocol (DNP) subDAOs, and more easily scale existing DNP
 subDAOs (e.g., IOT) by removing existing bottlenecks.
 
-With the deprecation of the Helium L1, changes to the [API](/blockchain/api) and
+With the deprecation of the Helium L1, changes to the API and
```

Comment:
#3,4 in the list below needs content updates.

Reply:
content added